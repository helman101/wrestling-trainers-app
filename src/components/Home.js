import React from 'react';

const Home = () => (
  <div>
    <h2>Wrestling Academy</h2>
    <img src="" alt="Academy logo" />
    <p>
      WA is a wrestling academy that dedicates its efforts to train new and promising wrestlers,
      our environment takes the profession seriously, if you decide to train with us,
      we can assure you that we will take you to
      the top as we have done before with other wrestlers,
      now our alumni, winners of several tournaments, return home to train new wrestling stars,
      will you be the next face of wrestling?
    </p>
  </div>
);

export default Home;
