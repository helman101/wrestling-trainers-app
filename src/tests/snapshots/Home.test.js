import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../components/Home';

it('renders correctly', () => {
  const quotePage = renderer
    .create(<Home />)
    .toJSON();
  expect(quotePage).toMatchSnapshot();
});