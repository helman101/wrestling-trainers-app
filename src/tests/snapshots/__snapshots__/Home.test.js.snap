// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt5"
>
  <div
    className="home margin0 dFlex justifyContentCenter mr2 alignItemsCenter"
  >
    <img
      alt="Academy logo"
      className="homeLogo mr1"
      src="logo.png"
    />
    <div>
      <h2
        className="title mb1"
      >
        Wrestling Academy
      </h2>
      <p
        className="mb1"
      >
        WA is a wrestling academy that dedicates its efforts to train new and promising wrestlers, our environment takes the profession seriously, if you decide to train with us, we can assure you that we will take you to the top as we have done before with other wrestlers, now our alumni, winners of several tournaments, return home to train new wrestling stars, will you be the next face of wrestling?
      </p>
    </div>
  </div>
</div>
`;
